diff --git a/node_modules/@sentry/utils/cjs/logger.js b/node_modules/@sentry/utils/cjs/logger.js
index 34f161d..7720afa 100644
--- a/node_modules/@sentry/utils/cjs/logger.js
+++ b/node_modules/@sentry/utils/cjs/logger.js
@@ -64,6 +64,7 @@ function makeLogger() {
       logger[name] = (...args) => {
         if (enabled) {
           consoleSandbox(() => {
+            console.log(`${PREFIX}[${name}]:`, ...args);
             worldwide.GLOBAL_OBJ.console[name](`${PREFIX}[${name}]:`, ...args);
           });
         }
