diff --git a/node_modules/@sentry/react-native/dist/js/tools/ModulesCollector.js b/node_modules/@sentry/react-native/dist/js/tools/ModulesCollector.js
index 8a6e51c..b65593a 100644
--- a/node_modules/@sentry/react-native/dist/js/tools/ModulesCollector.js
+++ b/node_modules/@sentry/react-native/dist/js/tools/ModulesCollector.js
@@ -88,6 +88,7 @@ class ModulesCollector {
             }
         }
         const map = JSON.parse((0, fs_1.readFileSync)(sourceMapPath, 'utf8'));
+        map.sources = undefined
         if (!map.sources || !Array.isArray(map.sources)) {
             utils_1.logger.error(`Modules not collected. No sources found in the source map (${sourceMapPath})!`);
             return;
